src/components/input/walletInput.tsx

```diff
@@ -172,6 +172,8 @@ export const WalletInput = React.forwardRef<HTMLTextAreaElement, WalletInputProp
          *               Hooks & Effects                 *
          *************************************************/
         useEffect(() => {
+            let isActive = true;
```

Comment:
Just a Simple ask, May I ask you to add a comment about the active flag for future reference? Thanks

Reply:
Sure good point, let me add a small comment